enlace/reports/attendance.php

```diff
@@ -199,9 +200,10 @@ function toggleAttendanceCheckboxes() {
                 }
                 $dosage_array = calculate_dosage($session_sqlsafe, null, $start_date_sqlsafe, $end_date_sqlsafe, $dropped_sqlsafe);
 ?>
-                <tr>
+                <tr <?php if ($dosage_array[4] != 0) { ?>class="unmarked"<?php } ?>>
```

Comment:
Hunh, so we extract specific meanings from $dosage_array via raw numerical indices?  If that's how we've been doing it, we don't need to fix it for this change, but it's too bad :-).